resources/cloudfront-function.go

```diff
@@ -0,0 +1,62 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudfront"
+	"github.com/rebuy-de/aws-nuke/v2/pkg/types"
+)
+
+type CloudFrontFunction struct {
+	svc   *cloudfront.CloudFront
+	name  *string
+	stage *string
+}
+
+func init() {
+	register("CloudFrontFunction", ListCloudFrontFunctions)
+}
+
+func ListCloudFrontFunctions(sess *session.Session) ([]Resource, error) {
+	svc := cloudfront.New(sess)
+	resources := []Resource{}
+
+	for {
+		resp, err := svc.ListFunctions(nil)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, item := range resp.FunctionList.Items {
+			resources = append(resources, &CloudFrontFunction{
+				svc:   svc,
+				name:  item.Name,
+				stage: item.FunctionMetadata.Stage,
+			})
+		}
+		return resources, nil
+	}
```

Comment:
Not sure if I missed anything, but this loop doesn't really make sense if you `return` after the first execution. Are you missing the actual pagination?

Reply:
You are right, I started implementing the pagination but fogot to actually add it on the way... I pushed an update adding it :)